zeppelin-web/src/app/notebook/paragraph/paragraph.controller.js

```diff
@@ -245,6 +245,7 @@ angular.module('zeppelinWebApp')
   $scope.cancelParagraph = function() {
     console.log('Cancel %o', $scope.paragraph.id);
     websocketMsgSrv.cancelParagraphRun($scope.paragraph.id);
+    $scope.paragraph.status = 'CANCELING';
```

Comment:
perhaps independent from this visual change, not everything is cancel-able from the interpreter? should there be a way to communicate that to the visual?